webapp/nuxt.config.js

```diff
@@ -162,7 +162,9 @@ module.exports = {
   // Give apollo module options
   apollo: {
     tokenName: 'human-connection-token', // optional, default: apollo-token
-    tokenExpires: 3, // optional, default: 7 (days)
+    cookieAttributes: {
+      expires: 3, // optional, default: 7 (days)
+    },
```

Comment:
is this related to a different deprecation warning that I'm not aware of? or how is this related?

Reply:
Err, yes, if you `yarn run dev` you see a deprecation warning of `nuxt-apollo` about the `tokenExpires`. They just added more options and want to put it into an object

Reply:
thanks for the explanation :)